common/utils/src/main/scala/org/apache/spark/SparkThrowableHelper.scala

```diff
@@ -34,7 +33,7 @@ private[spark] object ErrorMessageFormat extends Enumeration {
  */
 private[spark] object SparkThrowableHelper {
   val errorReader = new ErrorClassesJsonReader(
-    Seq(Utils.getSparkClassLoader.getResource("error/error-classes.json")))
+    Seq(getClass.getClassLoader.getResource("error/error-classes.json")))
```

Comment:
We should move part of the `Utils` as well. We should do that in a follow-up.

Reply:
Yes Utils is a bit big and I need to tackle that separately with a strategy